makeUSB.sh

```diff
@@ -69,6 +69,15 @@ unmountUSB() {
 # Trap kill signals (SIGHUP, SIGINT, SIGTERM) to do some cleanup and exit
 trap 'cleanUp' 1 2 15
 
+# Show help before checking for root
+case "$1" in
+	-h|--help)
+		showUsage
+		exit 0
+		;;
+esac
+shift
```

Comment:
Unconditional shift looks troublesome to me. Has this been tested?

Reply:
Yeah, I removed that.

Reply:
b14a4cc0616a5a6f08402e3d6bc30b76b5ea6171 i see. Cool! Thumbs up for 9300311fd0458847b79b6aa28638463487810d38.